README.md

```diff
@@ -36,7 +36,7 @@ which are both under active development.
 
 ## Requirements
 
-[Go](http://golang.org) 1.16 or newer.
+[Go](http://golang.org) 1.17 or newer.
```

Comment:
There's also an occurrence of this exact sentence in `docs/installation.go` and a checklist item in `docs/code_contribution_guidelines.md`.

Reply:
Addressed